<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-BVQ9VGC" failures="1" tests="5" name="org.TestngPrograms.ListenersHandle" time="28.911" errors="0" timestamp="02 Apr 2024 14:38:34 GMT" skipped="3">
  <testcase classname="org.TestngPrograms.ListenersHandle" name="Tc1" time="7.388"/>
  <testcase classname="org.TestngPrograms.ListenersHandle" name="Tc2" time="4.578">
    <skipped/>
  </testcase> <!-- Tc2 -->
  <testcase classname="org.TestngPrograms.ListenersHandle" name="Tc2" time="6.213">
    <skipped/>
  </testcase> <!-- Tc2 -->
  <testcase classname="org.TestngPrograms.ListenersHandle" name="Tc2" time="5.253">
    <skipped/>
  </testcase> <!-- Tc2 -->
  <testcase classname="org.TestngPrograms.ListenersHandle" name="Tc2" time="5.479">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at org.TestngPrograms.ListenersHandle.Tc2(ListenersHandle.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
]]>
    </failure>
  </testcase> <!-- Tc2 -->
</testsuite> <!-- org.TestngPrograms.ListenersHandle -->
